### PR TITLE
Da 362 version bump webpack refactoring

### PR DESCRIPTION
**Pull request by lokesh-couchbase**
## Short description of the changes
-

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -59,92 +60,105 @@
...
+        "@types/node": "^20.12.2",
         "@types/react": "^18.2.6",
         "@types/react-dom": "^18.2.4",
         "@types/tar": "^6.1.5",
         "@types/uuid": "^9.0.6",
-        "@types/vscode": "^1.63.1",
-        "@typescript-eslint/eslint-plugin": "^4.33.0",
-        "@typescript-eslint/parser": "^4.33.0",
-        "@vscode/test-electron": "^2.3.8",
+        "@types/vscode": "^1.87.0",
```
Comment (prajwal-pai77) [inline package-lock.json:68]: Shouldn't it be 1.63.1 it self?(see package.json)

**Review by prajwal-pai77 — APPROVED** [review]
Looks good

File: package-lock.json (hunk 1 above)
Comment (lokesh-couchbase) [reply package-lock.json]: nice find

**Comment by lokesh-couchbase** [comment]
Closing this PR for https://github.com/couchbaselabs/VSCode-Couchbase/pull/368

**Pull request closed without merging**